tests/framework/extensions/namespaces/getnamespaces.go

```diff
@@ -0,0 +1,23 @@
+package namespaces
+
+import (
+	"sort"
+
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+)
+
+// ListNamespaceNames is a helper which returns a sorted list of namespace names
+func ListNamespaceNames(steveclient *v1.Client) ([]string, error) {
+
+	namespaceList, err := steveclient.SteveType(NamespaceSteveType).List(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	namespace := make([]string, len(namespaceList.Data))
+	for idx, ns := range namespaceList.Data {
+		namespace[idx] = ns.GetName()
+	}
+	sort.Strings(namespace)
+	return namespace, nil
+}
```

Comment:
So we probably don't need these name list extensions anymore, that name list PR is merged. Could you rebase and use `List().Names()` from steve client?

Reply:
Updated for namespaces to make use of  List().Names()  that was recently added in our code.